installer/conf/sudoers

```diff
@@ -32,10 +32,17 @@ omsagent ALL=(ALL) NOPASSWD: /opt/microsoft/omsagent/ruby/bin/ruby /opt/microsof
 omsagent ALL=(ALL) NOPASSWD: /opt/microsoft/omsagent/ruby/bin/ruby /opt/microsoft/omsagent/bin/hdinsightmanifestreader.rb
 
 #run update management commands
-omsagent ALL=(ALL) NOPASSWD: /usr/bin/apt-get update 
-omsagent ALL=(ALL) NOPASSWD: /usr/bin/apt-get dist-upgrade * 
-omsagent ALL=(ALL) NOPASSWD: /usr/bin/apt-get upgrade *  
+omsagent ALL=(ALL) NOPASSWD: /usr/bin/apt-get dist-upgrade *
+omsagent ALL=(ALL) NOPASSWD: /usr/bin/apt-get update
+omsagent ALL=(ALL) NOPASSWD: /usr/bin/apt-get upgrade *
+omsagent ALL=(ALL) NOPASSWD: /usr/bin/apt-get * update
+omsagent ALL=(ALL) NOPASSWD: /usr/bin/apt-get * dist-upgrade
+omsagent ALL=(ALL) NOPASSWD: /usr/bin/apt-get * upgrade
+omsagent ALL=(ALL) NOPASSWD: /usr/bin/apt-get * install
```

Comment:
We have 3 commands perform install related actions. There is an ongoing discussion to determine which one can be removed.  I will update this pull request once we nail down the final list. 

apt-get -d -y install
apt-get -y --only-upgrade true install
apt-get -y --only-upgrade true -s install